_posts/0000-01-02-acepopo.md

```diff
@@ -0,0 +1,6 @@
+---
+layout:slide
+title:"welcome to my second slide"
+---
+your text
```

Comment:
### **I would like to make preprocessing in fmri but I donnnot know wwhat to do**